reference/constraints/Bic.rst

```diff
@@ -0,0 +1,107 @@
+Bic
+===
+
+.. versionadded:: 2.8
+    The Bic constraint was introduced in Symfony 2.8.
+
+This constraint is used to ensure that a value has the proper format of an
+`Business Identifier Code (BIC)`_. BIC is an internationally agreed means to
+uniquely identify both financial and non-financial institutions.
+
++----------------+-----------------------------------------------------------------------+
+| Applies to     | :ref:`property or method<validation-property-target>`                 |
++----------------+-----------------------------------------------------------------------+
+| Options        | - `message`_                                                          |
+|                | - `payload`_                                                          |
++----------------+-----------------------------------------------------------------------+
+| Class          | :class:`Symfony\\Component\\Validator\\Constraints\\Bic`              |
++----------------+-----------------------------------------------------------------------+
+| Validator      | :class:`Symfony\\Component\\Validator\\Constraints\\BicValidator`     |
++----------------+-----------------------------------------------------------------------+
+
+Basic Usage
+-----------
+
+To use the Bic validator, simply apply it to a property on an object that
+will contain a Business Identifier Code (BIC).
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        // src/Acme/AppBundle/Entity/Transaction.php
```

Comment:
Will the `Acme` part be removed in all other `reference/constraints/*.rst` files in 2.8?
Otherwise the `Bic.rst` would be inconsistent to the rest of the documentation.